src/resources/postcss/components/skeleton/_svgicon.pcss

```diff
@@ -0,0 +1,105 @@
+.tribe-common {
+	/* -----------------------------------------------------------------------------
+	 *
+	 * Component: SVGIcon
+	 *
+	 * Example:
+	 * <?php $this->template('components/icons/[name]') ?>
+	 *
+	 * ----------------------------------------------------------------------------- */
+
+	.tribe-common-c-svgicon--featured {
+		width: 8px;
+	}
+
+	.tribe-common-c-svgicon--recurring {
+		width: 12px;
+	}
+
+	.tribe-common-c-svgicon--search {
+		width: 16px;
+	}
+
+	.tribe-common-c-svgicon--location {
+		width: 10px;
+	}
+
+	.tribe-common-c-svgicon--list {
+
+		.tribe-common-c-svgicon__svg-fill {
+			fill: var(--color-icon-active);
+		}
+	}
+
+	.tribe-common-c-svgicon--month,
+	.tribe-common-c-svgicon--day,
+	.tribe-common-c-svgicon--photo,
+	.tribe-common-c-svgicon--week,
+	.tribe-common-c-svgicon--map {
+		height: 100%;
+		width: 100%;
+
+		path {
+			fill: var(--color-icon-active);
+		}
+	}
+
+	.tribe-common-c-svgicon--close,
+	.tribe-common-c-svgicon--close-alt {
+		path {
+			stroke: var(--color-icon-secondary);
+		}
+	}
+
+	.tribe-common-c-svgicon--no-map {
+		fill: none;
+
+		.tribe-common-c-svgicon__svg-fill {
+			fill: #111;
+		}
+
+		.tribe-common-c-svgicon__svg-stroke {
+			stroke: var(--color-icon-active);
+		}
+	}
+
+	.tribe-common-c-svgicon--phone,
+	.tribe-common-c-svgicon--mail,
+	.tribe-common-c-svgicon--website,
+	.tribe-common-c-svgicon--map-pin {
+		fill: none;
+
+		path {
+			stroke: var(--color-icon-active);
+		}
+	}
+	
+	.tribe-common-c-svgicon--caret-alt-left,
+	.tribe-common-c-svgicon--caret-alt-right,
+	.tribe-common-c-svgicon--caret-alt-down,
```

Comment:
also any reason why we have these icons named `caret-alt-{direction}`? if there isn't a really strong reason, could we change it to not include alt?

Reply:
@paulmskim the original `caret-left` and `caret-right` are used in another plugin `events-filterbar` (and maybe others?). Is it safe to update?

Reply:
They should be the same caret left and right as the ones used in filter bar.